bosh-stemcell/spec/support/cis.rb

```
@@ -1,11 +1,9 @@
-require 'set'
-
-$cis_test_cases = Set.new
+$cis_test_cases = Set.new # standard:disable Style/GlobalVars
```

RuboCop (1.86.0)

[convention] Do not introduce global variables.

(Style/GlobalVars)